docs/content/preview/explore/query-1-performance/explain-analyze.md

```diff
@@ -2,455 +2,274 @@
-is_valid    | boolean                        |           |          |
-created_at  | timestamp(6) without time zone |           | not null |
-updated_at  | timestamp(6) without time zone |           | not null |
-verified_at | timestamp without time zone    |           |          |
-
-Indexes:
-    "account_type_pkey" PRIMARY KEY, lsm (id HASH)
-```
-
-```sql
-yugabyte=# explain SELECT account_domains.* FROM account_type
-    WHERE account_type.account_id = 1234 ORDER BY account_type.url ASC;
-```
-
-```output
-QUERY PLAN
-----------------------------------------------
-Sort (cost=152.33..154.83 rows=1000 width=237)
-    Sort Key: url
-        Seq Scan on account_type (cost=0.00..102.50 rows=1000 width=237)
-            Filter: (account_id = 6873)
-```
-
-The query runs a sequential scan on the account_type table, so adding an index on `account_id` prevents the full scan of the table, as follows:
-
-```sql
-create index account_id on account_type(account_id);
-```
-
-With the new index, the query scans the index rather than the larger main table, significantly improving performance.
-
-```sql
-yugabyte=# explain SELECT account_type.* FROM account_type
-    WHERE account_type.account_id = 6873 ORDER BY account_type.url ASC;
-```
-
-```output
-QUERY PLAN
-----------------------------------------
-Sort (cost=5.39..5.42 rows=10 width=237)
-    Sort Key: url
-        Index Scan using account_id on account_type (cost=0.00..5.22 rows=10 width=237)
-            Index Cond: (account_id = 6873)
-
-Time: 71.757 ms
-Previous runtime: 460 ms
-```
+CREATE INDEX idx_value_3 ON kvstore(value ASC) INCLUDE(key);
+```
+
+Now if you run the same query, you should see,
+
+```yaml{linenos=inline, .nocopy}
+# EXPLAIN (ANALYZE, DIST, COSTS OFF) SELECT * FROM kvstore WHERE value LIKE 'ca%' ORDER BY VALUE;
+                                       QUERY PLAN
+-----------------------------------------------------------------------------------------
+ Index Only Scan using idx_value_3 on kvstore (actual time=0.884..0.935 rows=41 loops=1)
+   Index Cond: ((value >= 'ca'::text) AND (value < 'cb'::text))
+   Storage Filter: ((value)::text ~~ 'ca%'::text)
+   Storage Index Read Requests: 1
+   Storage Index Read Execution Time: 0.792 ms
+   Storage Index Rows Scanned: 41
+ Planning Time: 4.290 ms
+ Execution Time: 1.008 ms
+ Storage Read Requests: 1
+ Storage Read Execution Time: 0.792 ms
+ Storage Rows Scanned: 41
+ Catalog Read Execution Time: 3.824 ms
+ Storage Execution Time: 4.616 ms
+```
+
+Now, only 41 rows were scanned (via `Storage Rows Scanned: 41`) to retrieve 41 rows (via `rows=41`) and there was no sorting ionvolved as the data is stored sorted in the range index in `ASC` order. even though YugabyteDB has optimizations for reverse scans, if most of your queries are going to retrieve the data in `DESC` order, then it would be better to define the ordering as `DESC` when creating a range sharded table/index.
```

Comment:
```suggestion
Now, only 41 rows were scanned (via `Storage Rows Scanned: 41`) to retrieve 41 rows (via `rows=41`) and there was no sorting involved as the data is stored sorted in the range index in `ASC` order. Even though YugabyteDB has optimizations for reverse scans, if most of your queries are going to retrieve the data in `DESC` order, then it would be better to define the ordering as `DESC` when creating a range sharded table/index.
```